### PR TITLE
updated patch for new maxtext software version

### PR DESCRIPTION
**Pull request by kocchop**
updated patch for new maxtext software version

### CODE REVIEW

File: .github/container/test-maxtext.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ if [[ $? -ne 0 ]]; then
 fi
 
 # Default arguments
-MULTIPROCESS=false
+HARDWARE='tpu'
```
Comment (terrykong) [inline .github/container/test-maxtext.sh:38]: Do you mind leaving a comment that this is essentially a no-op? As it stands it kind of looks like we test tpu by default

**Review by terrykong — CHANGES_REQUESTED** [review]

File: .github/container/test-maxtext.sh (hunk 1 above)
Comment (kocchop) [reply .github/container/test-maxtext.sh]: Found another way around and also updated files @terrykong

**Review by terrykong — APPROVED** [review]

**Comment by olupton** [comment]
Just wondering: is there a reason this is being managed via a patch file in the repo instead of via external pr/branch like the others (rosetta pax etc.)?

**Comment by kocchop** [comment]
Hi @olupton, I am working towards that, will soon convert the patching similar to the rest

**Pull request merged**